### PR TITLE
Feature/fuentbit

### LINKED ISSUE #736: [FEATURE] fluentbit operator for logs
We need a role in the ace-box to deploy fluentbit for logs collection

### PR DESCRIPTION
**Pull request by IgnacioGoldman**
fluentbit collector for logs

### CODE REVIEW

File: user-skel/ansible_collections/ace_box/ace_box/roles/fluentbit/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    cmd: "sed -i 's,DT_INGEST_TOKEN_PLACEHOLDER,{{ fluent_bit_dt_token }},' {{ role_path }}/templates/fluent-bit-values.yaml"
+
+- name: Replace DT_INGEST_HOST_PLACEHOLDER for fluentbit file
+  shell:
+    cmd: "sed -i 's,DT_INGEST_HOST_PLACEHOLDER,{{ dt_tenant_no_protocol }},' {{ role_path }}/templates/fluent-bit-values.yaml"
+
+- name: Install Fluent Bit
+  kubernetes.core.helm:
+    name: fluent-bit
+    chart_ref: fluent/fluent-bit
```
Comment (drqc) [inline user-skel/ansible_collections/ace_box/ace_box/roles/fluentbit/tasks/main.yml:41]: we should set the fluentbit version for this install step. You can use as an example this 
 kubernetes.core.helm:
    name: "otel-demo"
    chart_ref: open-telemetry/opentelemetry-demo
    chart_version: "{{ otel_helm_chart_version }}"

**Review by drqc — CHANGES_REQUESTED** [review]
a couple of questions: is there a way to set the ed-ge connect version and also the fluentbit version should be static. I'm guessing you are moving the edge connect + fluent bit together in the same PR because are used together?

**Comment by kristofre** [comment]
@IgnacioGoldman this PR covers changes on easytrade, edgeconnect and fluentbit. Please separate those out, also please link the issue that the PR addresses/closes

**Review by drqc — APPROVED** [review]
the files lgtm, if you could link it to the issue so we have track in the relation between the issues and PR's

**Pull request merged**